addons/mail/static/tests/tours/mail_chatter_activity_test_tour.js

```diff
@@ -0,0 +1,36 @@
+/* @odoo-module */
+
+import { registry } from "@web/core/registry";
+
+
+registry.category("web_tour.tours").add("mail_activity_schedule_from_chatter", {
+    test: true,
+    steps: [
+        {
+            trigger: "button:contains('Activities')",
+        },
+        {
+            trigger: "input[id='summary_0']",
+            run: "text Play Mario Kart",
+        },
+        {
+            trigger: "button:contains('Mark as Done')",
+        },
+        {
+            trigger: '.o-mail-Message-body:contains("Play Mario Kart")',
```

Comment:
I would remove `-body`, as it's a technical detail (`.o-mail-Message:contains()`) is good enough